Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.3'
 
+#add simpleCov for Code Climate test coverage visibility
+gem 'simplecov', require: false, group: :test
```

Comment:
We already have group test, starting on the line `65`...i think it would be good idea to move this also there :)

Reply:
Excellent - thanks for the callout! I've moved it to the `group :test` block and ran Rspec again to make sure it didn't break anything :) Thanks and let me know if anything else is needed.